help/develop/javascript-development/datagrok-configuration.md

```diff
@@ -0,0 +1,39 @@
+# Datagrok configuration
+
+After you install `datagrok-tools` and run the first command (probably, to [generate a package]), you need to configure
+your environment.
+
+## Table of Contents
+
+* [Overview](#overview)
+* [Developer Keys](#developer-keys)
+
+## Overview
+
+The Grok CLI creates a configuration file under `%USERNAME%/.grok/config.yaml` to store the configurations for your
+server, developer keys, and other metadata that's used by Datagrok.
+
+You can configure this file from the terminal by running `grok config` and then entering the needed configurations. 
+Alternatively, you can change this file manually.
```

Comment:
Let's specify that `grok config` outputs the file location (and content), as it would be easier to navigate to it if users prefer to make changes manually.